spec/requests/api/v2/providers/suggest_provider_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe 'GET /suggest' do
+  let(:jsonapi_renderer) { JSONAPI::Serializable::Renderer.new }
+  let(:organisation) { create(:organisation) }
+  let(:provider1) { create(:provider, organisations: [organisation]) }
+  let(:provider2) { create(:provider, organisations: [organisation]) }
+  let(:next_recruitment_cycle) { find_or_create(:recruitment_cycle, :next) }
+  let(:provider3) { create(:provider, provider_name: provider2.provider_name, organisations: [organisation], recruitment_cycle: next_recruitment_cycle) }
```

Comment:
can we have a test for that as well please

 it 'searches for a particular provider for the user for the next recruitment cycle' do

Reply:
Done